mantidimaging/gui/windows/spectrum_viewer/spectrum_widget.py

```diff
@@ -104,22 +109,61 @@ def get_tof_range(self) -> tuple[int, int]:
         r_min, r_max = self.range_control.getRegion()
         return int(r_min), int(r_max)
 
-    def random_colour_generator(self) -> tuple[int, int, int]:
+    def random_colour_generator(self) -> tuple[int, int, int, int]:
         """
         A random colour generator to colour ROIs boarders.
         Generates colours that are easy to see for colour blind people if colour_blind_friendly is True.
         By default colour_blind_friendly is set to False
 
-        @return: A random colour in RGB format. (0-255, 0-255, 0-255)
+        @return: A random colour in RGB format. (0-255, 0-255, 0-255, 0-255)
         """
-        accessible_colours = [(255, 194, 10), (12, 123, 220), (153, 79, 0), (64, 176, 166), (230, 97, 0), (93, 58, 155),
-                              (26, 255, 26), (254, 254, 98), (211, 95, 183), (220, 50, 43)]
+        accessible_colours = [(255, 194, 10, 255), (12, 123, 220, 255), (153, 79, 0, 255), (64, 176, 166, 255),
+                              (230, 97, 0, 255), (93, 58, 155, 255), (26, 255, 26, 255), (254, 254, 98, 255),
+                              (211, 95, 183, 255), (220, 50, 43, 255)]
         if self.colour_index == len(accessible_colours):
             self.colour_index = 0
         colour = accessible_colours[self.colour_index]
         self.colour_index += 1
         return colour
```

Comment:
I'll update the docstring. thanks for noticing that 👍 
Because the colours are in tuples to be the correct type to pass to PyQt, they are immutable. I could set a variable called alpha to include in each tuple:
```Python
alpha = 255
accessible_colours = [(255, 194, 10, 255), (12, 123, 220, 255), (153, 79, 0, 255), (64, 176, 166, 255),
     230, 97, 0, 255), (93, 58, 155, 255), (26, 255, 26, 255), (254, 254, 98, 255), (211, 95, 183, 255),
    (220, 50, 43, 255)]
```
 or convert to a list and back into a tuple:

```Python
accessible_colours = [(255, 194, 10), (12, 123, 220), (153, 79, 0), (64, 176, 166), (230, 97, 0),
    (93, 58, 155), (26, 255, 26), (254, 254, 98), (211, 95, 183), (220, 50, 43)]
if self.colour_index == len(accessible_colours):
    self.colour_index = 0
colour = list(accessible_colours[self.colour_index])
colour.append(255)
self.colour_index += 1
return tuple(colour)
```

Reply:
The second version. But you shouldn't need to convert to a list, you can add 2 tuples to make a new one, so
`return colour + (255,)`
should work with even `colour` as a tuple.